dast/diff.py

```diff
@@ -0,0 +1,336 @@
+"""
```

Comment:
I will read through this properly to understand but it feels quite dangerous to do this... If any of the internal features of `DeepDiff` changes this will likely break, so we'll have to pin the version. In principle that's fine but we should see if there are simpler ways to do it, for example through `iterable_compare_func`

Reply:
I wasn't feeling right neither when I did in this way, and yep we should pin the version.
I am totally open to find a better solution, the problem with `iterable_compare_func` is that acts locally: takes in input two elements of the iterables and returns if they are a match or not, so it does depend on ordering! (Unless you just match exact matching). Instead this approach leverages `_get_most_in_common_pairs_in_iterables`, which evaluates globally